src/App.tsx

```diff
@@ -52,7 +52,7 @@ const App: React.FC = () => (
           <Route path="/constitution/toc" component={ConstitutionTOC} exact={true} />
           <Route path="/constitution/full" component={Constitution_Full} exact={true} />
           <Route path="/constitution/full/:id+" component={Constitution_Full} exact={true} />
-          <Route path="/constitution/provision/:id+" component={Constitution} exact={true} />
+          <Route path="/constitution/toc/:id+" component={Constitution} exact={true} />
```

Comment:
Nice! `/constitution/full` still goes wrong. So how about this:

Make the TOC view `/constitution`, so then everything is underneath that (eg. `/constitution/full`, `/constitution/provision/:id+` etc.)

Reply:
So the primary Constitution tab view is `/constitution`